content/en/docs/bmf/api/decode_module/_index.md

```diff
@@ -8,9 +8,17 @@ weight: 4
 This is a module capability discrption about BMF build-in decoder. The module can be used by BMF API such as  [bmf.decode()](https://babitmf.github.io/docs/bmf/api/api_in_python/transcode_functions/#decode)  by providing json style "option" to config such as the 3rd parameter below:
 
 
-```
+```py
 bmf.decode(
-    {'input_path': input_video_path}
+    {
+        "input_path": input_path,
+        "video_params": {
+            "hwaccel": "cuda",
```

Comment:
How about using other more common params such as codec, width, height, crf, preset. The hwaccel config example is showed below.

Reply:
Got it. I'll add more details for the code example